bears/general/CPDBear.py

```diff
@@ -0,0 +1,40 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
+    executable = 'run.sh'
```

Comment:
they are in the other commit, i thought the bears are independent, now i understand.